modules/otel-config-instrumentation.adoc

```diff
@@ -98,6 +98,94 @@ spec:
 
 |===
 
+[id="otel-instrumentation-options_exporter_{context}"]
+== Exporter configuration
+
+Although the `Instrumentation` custom resource supports setting up one or more exporters per signal, the auto-instrumentation configures only the OTLP Exporter. Due to this, you must configure the endpoint to point to the OTLP Receiver on the Collector. You can do so by using a config map as well as a Secret for the TLS and CA certificates.
```

Comment:
I would remove "You can do so by using a config map as well as a Secret for the TLS and CA certificates."

It does not make sense. The config map or secret is used only for TLS.

Reply:
We probably had a misunderstanding in https://github.com/openshift/openshift-docs/pull/83387/files#r1851822665 then.